### PR TITLE
Update glfw package to 3.3.4

### PR DESCRIPTION
**Pull request by alex-tdrn**
I've followed [this guide](https://hunter.readthedocs.io/en/latest/creating-new/update.html) step by step carefully. **[Yes]**

### CODE REVIEW

File: cmake/projects/glfw/hunter.cmake [hunk 1]
```diff
@@ -90,6 +90,28 @@ hunter_add_version(
     533debc50f12ce147d54d210408c7e33654f10a4
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    glfw
+    VERSION
+    "3.3.4-p0"
```
Comment (rbsheth) [inline cmake/projects/glfw/hunter.cmake:97]: Can you remove this version, if it isn't useful?

Comment (alex-tdrn) [reply cmake/projects/glfw/hunter.cmake:97]: I've deleted the p1 tag, and pointed the p0 tag to the latest glfw commit on the 3.3.4 branch now

**Comment by NeroBurner** [comment]
@alex-tdrn could you rebase and squash the changes

**Review by rbsheth — APPROVED** [review]

**Comment by rbsheth** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.23.317

**Pull request merged**